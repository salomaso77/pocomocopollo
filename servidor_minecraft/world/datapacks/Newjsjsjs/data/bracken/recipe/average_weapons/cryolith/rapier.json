{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:raiser_armor_trim_smithing_template",
	"base": "minecraft:iron_sword",
	"addition": "minecraft:polar_bear_spawn_egg",
	"result": {
		"id": "minecraft:iron_sword",
		"components": {
			"minecraft:max_damage": 200,
			"minecraft:item_name": "Cryolith Rapier",
			"minecraft:item_model": "bracken:average_weapons/cryolith/rapier",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 3,
	"factor": 3
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_sword"
},
			"minecraft:enchantments": {
				"minecraft:vanishing_curse": 1,
				"bracken:freezing_fix": 1,
				"bracken:fragility_curse": 1
			},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "10 Attack Damage","color":"dark_green","italic":false},
				{"translate": "2 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:cryolith_rapier",
					"amount": 9,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:cryolith_rapier",
					"amount": -2,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}