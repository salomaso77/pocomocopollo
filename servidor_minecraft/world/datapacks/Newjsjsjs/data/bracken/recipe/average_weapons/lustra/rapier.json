{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:raiser_armor_trim_smithing_template",
	"base": "minecraft:golden_sword",
	"addition": "minecraft:sheep_spawn_egg",
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:max_damage": 3000,
			"minecraft:enchantment_glint_override": true,
			"minecraft:item_name": "Lustra Rapier",
			"minecraft:item_model": "bracken:average_weapons/lustra/rapier",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 3,
	"factor": 3
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_sword"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "4 Attack Damage","color":"dark_green","italic":false},
				{"translate": "2 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:lustra_rapier",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:lustra_rapier",
					"amount": -2,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}