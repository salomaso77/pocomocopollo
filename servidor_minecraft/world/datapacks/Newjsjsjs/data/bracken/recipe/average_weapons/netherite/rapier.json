{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:raiser_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:netherite_ingot",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Netherite Rapier",
			"minecraft:item_model": "bracken:average_weapons/netherite/rapier",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 3,
	"factor": 3
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_sword"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "8 Attack Damage","color":"dark_green","italic":false},
				{"translate": "2 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:netherite_rapier",
					"amount": 7,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:netherite_rapier",
					"amount": -2,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}