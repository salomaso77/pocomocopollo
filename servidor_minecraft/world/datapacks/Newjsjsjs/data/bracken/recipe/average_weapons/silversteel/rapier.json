{
	"type": "minecraft:smithing_transform",
	"template": "minecraft:raiser_armor_trim_smithing_template",
	"base": "minecraft:netherite_sword",
	"addition": "minecraft:wolf_spawn_egg",
	"result": {
		"id": "minecraft:netherite_sword",
		"components": {
			"minecraft:item_name": "Silversteel Rapier",
			"minecraft:enchantments": {
				"bracken:silvered_fix": 1
			},
			"minecraft:enchantment_glint_override": false,
			"minecraft:item_model": "bracken:average_weapons/silversteel/rapier",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 3,
	"factor": 3
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_sword"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "9 Attack Damage","color":"dark_green","italic":false},
				{"translate": "2 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:silversteel_rapier",
					"amount": 8,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:silversteel_rapier",
					"amount": -2,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}