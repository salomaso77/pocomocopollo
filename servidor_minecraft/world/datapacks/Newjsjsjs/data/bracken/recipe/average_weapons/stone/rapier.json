{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		" # ",
		"## ",
		" / "
	],
	"key": {
		"#": "#minecraft:stone_tool_materials",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:stone_sword",
		"components": {
			"minecraft:item_name": "Stone Rapier",
			"minecraft:item_model": "bracken:average_weapons/stone/rapier",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 3,
	"factor": 3
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_sword"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "5 Attack Damage","color":"dark_green","italic":false},
				{"translate": "2 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:stone_rapier",
					"amount": 4,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:stone_rapier",
					"amount": -2,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}