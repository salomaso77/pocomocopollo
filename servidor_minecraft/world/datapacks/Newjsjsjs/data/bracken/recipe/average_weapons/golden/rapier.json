{
	"type": "minecraft:crafting_shaped",
	"category": "equipment",
	"pattern": [
		" # ",
		"## ",
		" / "
	],
	"key": {
		"#": "minecraft:gold_ingot",
		"/": "minecraft:stick"
	},
	"result": {
		"id": "minecraft:golden_sword",
		"components": {
			"minecraft:item_name": "Golden Rapier",
			"minecraft:item_model": "bracken:average_weapons/golden/rapier",
			"minecraft:blocks_attacks": {
  "block_delay_seconds": 0,
  "disable_cooldown_scale": 0.1,
  "item_damage": {
	"threshold": 0,
	"base": 3,
	"factor": 3
  },
  "block_sound": "minecraft:block.anvil.place",
  "disabled_sound": "minecraft:item.axe.scrape",
  "bypassed_by": "#bracken:bypasses_sword"
},
			"minecraft:lore": [
				{"translate":"","color":"white","italic":false},
				{"translate":"When in Main Hand:","color":"gray","italic":false},
				{"translate": "4 Attack Damage","color":"dark_green","italic":false},
				{"translate": "2 Attack Speed","color":"dark_green","italic":false}
			],
			"minecraft:attribute_modifiers":[{
					"type": "minecraft:attack_damage",
					"id": "bracken:golden_rapier",
					"amount": 3,
					"operation": "add_value",
					"slot": "mainhand"
				},{
					"type": "minecraft:attack_speed",
					"id": "bracken:golden_rapier",
					"amount": -2,
					"operation": "add_value",
					"slot": "mainhand"
				}],
				"minecraft:tooltip_display": {
        "hidden_components": [
          "minecraft:attribute_modifiers"
        ]
      }
		}
	}
}